src/mlpack/tests/frankwolfe_test.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file frankwolfe_test.cpp
+ * @author Chenzhe Diao
+ *
+ * Test file for Frank-Wolfe optimizer.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/core/optimizers/fw/frank_wolfe.hpp>
+#include <mlpack/core/optimizers/fw/constr_lpball.hpp>
+#include <mlpack/core/optimizers/fw/update_span.hpp>
+#include <mlpack/core/optimizers/fw/update_classic.hpp>
+#include <mlpack/core/optimizers/fw/func_sq.hpp>
+#include <mlpack/core/optimizers/fw/test_func_fw.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "test_tools.hpp"
+
+using namespace std;
+using namespace arma;
+using namespace mlpack;
+using namespace mlpack::optimization;
+
+BOOST_AUTO_TEST_SUITE(FrankWolfeTest);
+
+BOOST_AUTO_TEST_CASE(OMPTest)
```

Comment:
Can you add a comment for each test, a single sentence is often enough, no details needed.